### ISSUE TITLE
Shared config files between Linux X Windows

### ISSUE THREAD

**Issue by tuxtergames** [coldfix/steam-acolyte#26]
Hello,

Is it possible share configuration files between Linux X Windows or vice versa, to avoid typing password every time?
Dual boot just socks, nice job man, simple and efficient program.

**Comment by coldfix** [comment]
Hi, not sure. Is it necessary? There should normally be no problem in having separate active logins on both OS at the same time.

**Comment by tuxtergames** [comment]
> Hi, not sure. Is it necessary? There should normally be no problem in having separate active logins on both OS at the same time.

I've some machines here and I use to dual boot then, and its just a curiosity about acolyte.

**Comment by coldfix** [comment]
Ok.

it's not supported. You have to login each user on each steam installation, i.e. for each different machine or OS. That's how I do it as well. But you should not have to reenter your password on either OS once you logged in -- unless you stop using it for some weeks. Logging in on one platform does not log you out on others.

It might in theory be possible to transmit the login from one platform to the next, but that would open up a whole can of synchronization headaches because logging in always invalidates the previous login token within the same series. That means:

- acolyte would always need to check both installation folders for new version of the login token, i.e. your linux partition needs to be mounted in windows and vice versa, and acolyte needs to be configured the same way on both OS.
- One way to address this might be to synchronize login tokens through some network synchronization server, but now you would also have to worry to guard that network service against attackers, and this would not fix the following issues either:
- if you start steam without acolyte on either OS, you might be using a login token that was already "used up" by the other OS, so you'll have to reenter your password/2fa
- if you started and logged in steam without acolyte on both OS, both login tokens change. Which one should acolyte adopt as the "correct one"?

In summary, I think it's currently not viable.

Best, Thomas

**Comment by coldfix** [comment]
I do think the network service option could be very convenient if you don't care about ever starting steam without acolyte. But it's *a lot* of work that I cannot currently provide, and it would have to be very well-designed to guard against unauthorized access. Another question would be where to host.

**Comment by tuxtergames** [comment]
> I do think the network service option could be very convenient if you don't care about ever starting steam without acolyte. But it's _a lot_ of work that I cannot currently provide, and it would have to be very well-designed to guard against unauthorized access. Another question would be where to host.

tks for the answer, now its clear....
